test/functional/buildAndPackage/build.xml

```diff
@@ -54,10 +54,16 @@
             <fileset dir="${build}"/>
             <fileset dir="${src}/../" includes="*.properties,*.xml"/>
         </jar>
-        <copy todir="${DEST}">
```

Comment:
Aren't other .xml and .mk files expected to be packaged?

Reply:
Good question.
build.xml is not needed since it only is used in this "ant" step.

.mk are auto generated and i did a check only 3 xml files are in the ${src}/.../ 

the tricky part is the playlist.xml
see https://ci.adoptopenjdk.net/job/Grinder/5555/console when i turned on ant -d --verbose with some debug step in target dist
```
16:23:14       [exec] Current OS is Linux
16:23:14       [exec] Executing 'ls' with arguments:
16:23:14       [exec] '-lat'
16:23:14       [exec] '/home/jenkins/workspace/Grinder/aqa-tests/TKG/../../jvmtest/functional/buildAndPackage'
16:23:14       [exec] 
16:23:14       [exec] The ' characters around the executable and arguments are
16:23:14       [exec] not part of the command.
16:23:14       [exec] total 40
16:23:14       [exec] drwxr-xr-x    2 jenkins  jenkins       4096 Aug 31 14:00 .
16:23:14       [exec] -rw-r--r--    1 jenkins  jenkins      30307 Aug 31 14:00 BuildAndPackagingTests.jar
16:23:14       [exec] drwxr-xr-x    3 jenkins  jenkins       4096 Aug 31 14:00 ..
```
so before the copy, $DEST only got BuildAndPakcagingTests.jar then did 
```
16:23:14       [exec] Executing 'cp' with arguments:
16:23:14       [exec] '/home/jenkins/workspace/Grinder/aqa-tests/functional/buildAndPackage/src/../testng.xml'
16:23:14       [exec] '/home/jenkins/workspace/Grinder/aqa-tests/TKG/../../jvmtest/functional/buildAndPackage'
```
with only one file testng.xml, but later 
`16:23:30  Parsing /home/jenkins/workspace/Grinder/aqa-tests/TKG/../functional/buildAndPackage/playlist.xml`
so something must did the copy (probably some code in TKC/src/org/testKitGen)

Reply:
made a change to copy up all the xml files

test run in grinder https://ci.adoptopenjdk.net/job/Grinder/5561/console 
```
18:07:05      [apply] Current OS is Linux
18:07:05      [apply] build.xml added as build.xml doesn't exist.
18:07:05      [apply] playlist.xml added as playlist.xml doesn't exist.
18:07:05      [apply] testng.xml added as testng.xml doesn't exist.
18:07:05      [apply] Executing 'cp' with arguments:
18:07:05      [apply] '/home/jenkins/workspace/Grinder/aqa-tests/functional/buildAndPackage/build.xml'
18:07:05      [apply] '/home/jenkins/workspace/Grinder/jvmtest/functional/buildAndPackage/build.xml'
18:07:05      [apply] 
18:07:05      [apply] The ' characters around the executable and arguments are
18:07:05      [apply] not part of the command.
18:07:05      [apply] Executing 'cp' with arguments:
18:07:05      [apply] '/home/jenkins/workspace/Grinder/aqa-tests/functional/buildAndPackage/playlist.xml'
18:07:05      [apply] '/home/jenkins/workspace/Grinder/jvmtest/functional/buildAndPackage/playlist.xml'
18:07:05      [apply] 
18:07:05      [apply] The ' characters around the executable and arguments are
18:07:05      [apply] not part of the command.
18:07:05      [apply] Executing 'cp' with arguments:
18:07:05      [apply] '/home/jenkins/workspace/Grinder/aqa-tests/functional/buildAndPackage/testng.xml'
18:07:05      [apply] '/home/jenkins/workspace/Grinder/jvmtest/functional/buildAndPackage/testng.xml'
18:07:05      [apply] 
18:07:05      [apply] The ' characters around the executable and arguments are
18:07:05      [apply] not part of the command.
18:07:05      [apply] Applied cp to 3 files and 0 directories.
```